httpony/tests/test_server.py

```diff
@@ -8,6 +8,8 @@
 class TestServer(unittest.TestCase):
 
     @mock.patch('httpony.server.run_simple')
+    @mock.patch('httpony.server.hostname')
```

Comment:
`mock` can patch imported modules. It can't patch arbitrary variables inside functions so this won't work. As I noted above, changing the signature of `main` will make it so this mocking won't be necessary anyway.